new-note-namer/new-note-namer.qml

```diff
@@ -41,7 +49,11 @@ QtObject {
     script.log(note.fileCreated)
     script.log(note.fileLastModified)
 
-    return "# " + newName;
+    if (underlineHeading){
+          return newName + "\n========================"
+    } else {
+          return "# " + newName;
```

Comment:
Can you please adapt the amount of whitespace?

Reply:
I fixed my portion of the code. The original script is not properly consistent with its use of whitespace in the first place. Please let me know if I shall unify the whitespacing throughout the whole script.

Reply:
The whole script is a mess, you are right. Please do so in another PR.